### DISCUSSION TITLE
[Enhancement/Request] Location Specialized Command Prefix

### DISCUSSION THREAD

**Discussion by mercurialmusic** [Zrips/CMI#324] [Feature Requests]
Description of issue or feature request:
I've seen with a few other issues here how this might be useful, but I'm suddenly in a bind and need it myself. We're setting up some animated events where a player clicks an object (IC) that sets off a structure block and command block chain that animates the reveal of a secret entrance, etc. and currently have to use the execute command since Minecraft's /setblock command sucks and doesn't let you specify a world. The thing with execute, though, is that sometimes it works with these and sometimes the IC picks a random player who could be in another world completely, so I end up with floating redstone blocks in our end. Anyway, that's a long intro to a short request:

A specialized command prefix targeting a specific world and coords would, I think, be extremely useful for a lot of your commands. For example: atLocation:[world:x,y,z]! would allow us to execute the command including Minecraft's own /setblock command, at the specified location. Cool, right? :)

Cmi Version (using`/cmi version`): 7.7.1.2
Server Type (Spigot/Paperspigot/etc): Spigot 1.12.2 [Latest]
Server Version (using `/ver`):

**Comment by mercurialmusic** [comment]
Hmmm. Ok, I realized that using the [playerName] placeholder in the execute command instead of the @ p targeter seems to be more accurate, so I've switched to that and it solved my original problem, but I still see how this can be useful. Dunno. Definitely not a priority thing for me, but maybe if it's an easy thing to do.

**Comment by MCTyler** [comment]
I thought he added this feature???? Is this only for the Count command? (Although I have not been able to get it to work.) https://github.com/Zrips/CMI/issues/304 and https://github.com/Zrips/CMI/issues/279 (closed)
